www/docs/main/example-apps.mdx

```diff
@@ -0,0 +1,183 @@
+---
+id: example-apps
+title: Example Apps
+sidebar_label: Example Apps
+slug: /example-apps
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+:::info
+Some of the examples on this page need to be upgraded to V10. Please feel free to help us update!
+:::
+
+---
+
+### Next.js starter with Prisma, E2E testing, and ESLint (Recommended)
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+
+```bash
+npx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter trpc-prisma-starter
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+
+```sh
+yarn create next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter trpc-prisma-starter
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```sh
+pnpx create-next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-starter trpc-prisma-starter
+```
+
+  </TabItem>
+</Tabs>
+
+- Example deploy: [nextjs.trpc.io](https://nextjs.trpc.io)
+- [Source code](https://github.com/trpc/examples-next-prisma-starter)
+
+---
+
+### zART-stack example (zero-API, React, TypeScript)
```

Comment:
This project is deprecated and won't be updated to v10. Maybe we should link https://github.com/t3-oss/create-t3-turbo instead?

<img width="822" alt="CleanShot 2022-11-17 at 12 48 05@2x" src="https://user-images.githubusercontent.com/51714798/202438498-06f76724-fb1c-4365-93f2-0438d995c39d.png">

Reply:
Yes! feel free to edit @juliusmarminge